### PR TITLE
Rename team to data center, #23275

### PR DESCRIPTION
**Pull request by patriknw**
Refs #23275

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM. Quickly, merge it before it conflicts with anything! ;)

**Review by ktoso — APPROVED** [review]
LGTM, looks good

File: akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala [hunk 1]
```diff
@@ -18,18 +18,18 @@ import scala.concurrent.duration.FiniteDuration
 import akka.japi.Util.immutableSeq
 
 object ClusterSettings {
-  type Team = String
+  type DataCenter = String
```
Comment (jrudolph) [inline akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala]: Could we make it a value class (`extends AnyVal`) to get real type checking here?

**Review by jrudolph — APPROVED** [review]
LGTM. The docs will also need to be changed.

File: akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala (hunk 1 above)
Comment (ktoso) [inline akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala]: If we can pull it off I'm for it (or even a normal class)

Comment (johanandren) [inline akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala]: The reason we didn't was Java interop

**Comment by patriknw** [comment]
most docs is in that other PR, so I'll do that when I do another iteration

File: akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala (hunk 1 above)
Comment (jrudolph) [inline akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala]: I see.

**Pull request merged**